packages/sources/wrapped/src/endpoint/deposits.ts

```diff
@@ -0,0 +1,51 @@
+import { Requester, Validator, AdapterError } from '@chainlink/ea-bootstrap'
+import { Config, ExecuteWithConfig, InputParameters, AxiosResponse } from '@chainlink/types'
+
+export const supportedEndpoints = ['deposits']
+
+export interface ResponseSchema {
+  [token: string]: Addresses[]
+}
+
+export type Addresses = {
+  address: string
+}
```

Comment:
Could we just call this `Address` or `AddressObject`? It's a little confusing that you have the type `Addresses` which has a single address, then multiple places with the type `Addresses[]`.